### ISSUE TITLE
feat: Add support for `fontFamily` in `MiraiTheme`

### ISSUE THREAD

**Issue by RodolfoSilva** [Securrency-OSS/mirai#118]
**Description**

We need to provide support for custom `fontFamily` in the `MiraiTheme`.

**Requirements**

- [ ] Allow set `fontFamily`
- [ ] Allow set `fontFamilyFallback`

**Additional Context**

Maybe we could use Google Font to archive that or use fonts placed in the asset folder.
Also we should provide a way to define the `fontFamilyFallback`.

```
{
   'fontFamily' : 'Inter',
   'fontFamilyFallback': ['Roboto']
}
```

**Comment by loud-at-heart** [comment]
@RodolfoSilva can I take up this task

**Comment by RodolfoSilva** [comment]
Yes

**Comment by i-asimkhan** [comment]
Any update on this @loud-at-heart

**Comment by loud-at-heart** [comment]
Will provide an update by end of this week got stuck in some other work

**Comment by i-asimkhan** [comment]
Hey guys, do you mean Google fonts usage here by using [google fonts package](https://pub.dev/packages/google_fonts)  @divyanshub024  @RodolfoSilva

**Comment by RodolfoSilva** [comment]
Hello @i-asimkhan, I mean fonts from the assets folder.

**Comment by divyanshub024** [comment]
Closed by #205
